core/kazoo_bindings/src/kazoo_bindings.erl

```diff
@@ -819,13 +818,14 @@ bindings(Routing) ->
                              }=Bind, Acc) ->
                       case Binding =:= Routing
                           orelse matches(BParts, RoutingParts)
-                          orelse matches(RoutingParts, BParts)
```

Comment:
that's why i asked you to add same logic as in `fold`and `map `.
`bindings` was first introduced as `matches/1` mostly for debugging purposes. it had the same logic applied as the `orelse` changes introduced by that time.